src/view/events-item-view.js

```diff
@@ -1,23 +1,26 @@
 import { createElement } from '../render.js';
+import {formatStringToShortDate,formatStringShorTime, callcDate} from '../utils.js';
 
-function createEventsItemViewTemplate(){
+
+function createEventsItemViewTemplate(data){
+  const {point, pointDestination, pointOffers} = data;
   return(`            <li class="trip-events__item">
   <div class="event">
-    <time class="event__date" datetime="2019-03-18">MAR 18</time>
+    <time class="event__date" datetime="${point.dateFrom}">${formatStringToShortDate(point.dateFrom)}</time>
     <div class="event__type">
       <img class="event__type-icon" width="42" height="42" src="img/icons/taxi.png" alt="Event type icon">
```

Comment:
Иконку тоже поменять на ту, что соответствует типу
```suggestion
      <img class="event__type-icon" width="42" height="42" src="img/icons/${point.type}.png" alt="Event type icon">
```